src/main/java/com/study/realworld/user/presentation/model/UserLoginRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.study.realworld.user.presentation.model;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.study.realworld.user.application.model.UserLoginModel;
+
+import javax.validation.constraints.NotBlank;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.As.WRAPPER_OBJECT;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME;
+
+/**
+ * @author Jeongjoon Seo
+ */
+@JsonTypeName("user")
+@JsonTypeInfo(include = WRAPPER_OBJECT, use = NAME)
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserLoginRequest {
+
+    @NotBlank
+    private String email;
+
+    @NotBlank
+    private String password;
+
+    public UserLoginModel toModel() {
```

Comment:
```suggestion
    public UserLoginModel toUserLoginModel() {
```
네이밍관련해서 한가지 이야기해보면 좋을 것 같은데요 😃 
request -> Model의 경우 `toModel()`로 변환
Model -> entity의 경우 `toUser()`로 되어있어요 :)

일관성을 고려한다면 request to Model을 `toUserLoginModel()`로 변경하는것이 더 좋지않을까? 해서 코멘트 납깁니다~!

Reply:
이거 일부러 명시 안하고 모델로 변환해주는거는 다 toModel()로 쓰려고 합니다
RegisterModel로의 변환에서도 `request.toModel()`로 쓰고 있구요!
음 뭐랄까 이것도 관심사의 분리라고도 할 수 있으려나요? 그냥 request한테 model로 변경해서 넘겨라~ 이런 느낌...?